src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -122,6 +135,13 @@ protected void execute(final Event e) {
 					if (script != null)
 						scriptName = script.getConfig().getFileName();
 				}
+
+				if (mark == 1) {
+					Skript.warning("[" + scriptName + "] " + message);
+				}
+				if (mark == 2) {
+					Skript.error("[" + scriptName + "] " + message);
+				}
 				Skript.info("[" + scriptName + "] " + message);
```

Comment:
Again, should be a switch statement, probably.

Reply:
What do you mean?

Reply:
do you know what a switch statement is?